opencontrail/config.sls

```diff
@@ -154,7 +154,7 @@ publisher_init:
     - service: opencontrail_config_services
 {%- endif %}
 
-{%- if not grains.get('virtual_subtype', None) == "Docker" %}
+{%- if not common.vendor == "juniper" or not grains.get('virtual_subtype', None) == "Docker" %}
 
 /etc/contrail/supervisord_config_files/ifmap.ini:
```

Comment:
Put condition here, if juniper contrail then do not absent this file. Or maybe we can move ifmap for opencontrail under supervisor as well.